app/services/participant_uplift_aggregator.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "has_di_parameters"
+
+class ParticipantUpliftAggregator
+  include HasDIParameters
+
+  def call(event_type: :started)
+    recorder.send(params[event_type], lead_provider)
+  end
+
+private
+
+  def default_params
+    {
+      recorder: ParticipantDeclaration,
+      started: :count_active_uplift_for_lead_provider,
```

Comment:
I don't think we need this configurability now and we should simplify this class down to something smaller and easier to follow

Reply:
Now as in "not yet", or now as in there's something that can replace it that is equally as good or better?